Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "egui_logger"
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
Isn't this a breaking change, since it brings in a new egui version?
Which means the minor version should be updated (0.8.0)

Reply:
Yeah would make more sense